tests/functional/Parser/Fixtures/Assignments/issue_1.typoscript.print

```diff
@@ -1,5 +1,6 @@
 config.tx_extbase {
     view {
+# Configure where to look for widget templates
```

Comment:
Picking up on my earlier comment concerning indentation; at least in my preference, this comment should have the same indentation level as the following line.

Reply:
Absolutely. Works nicely.